addons/analytic/models/analytic_line.py

```diff
@@ -1,8 +1,10 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+from dateutil.relativedelta import relativedelta
+
```

Comment:
I know I wrote that blank line, but `dateutil` is actually not a basic module but a third party one, so it should be in the same group as `lxml`
```suggestion
```